boards/BMS/Inc/App/App_Accumulator.h

```diff
@@ -16,18 +31,16 @@
 #define MAX_CELL_VOLTAGE (4.2f)
 #define MIN_CELL_VOLTAGE (3.0f)
 
+// Discharge Parameters
+#define CELL_VOLTAGE_DISCHARGE_WINDOW_UV (600U)
+
 struct Accumulator;
 
 struct Accumulator *App_Accumulator_Create(
     bool (*config_monitoring_chip)(void),
     bool (*write_cfg_registers)(void),
     bool (*start_voltage_conv)(void),
-    bool (*read_cell_voltages)(void),
-    float (*get_min_cell_voltage)(uint8_t *, uint8_t *),
-    float (*get_max_cell_voltage)(uint8_t *, uint8_t *),
-    float (*get_segment_voltage)(AccumulatorSegments_E),
-    float (*get_pack_voltage)(void),
-    float (*get_avg_cell_voltage)(void),
+    bool (*read_cell_voltages)(uint16_t[ACCUMULATOR_NUM_OF_SEGMENTS][ACCUMULATOR_NUM_OF_SERIES_ELEM_PER_SEGMENT]),
```

Comment:
Didn't know you could pass arrays like this in C -> is this still the same under the hood as just passing by pointer? Or will the array be copied?

Reply:
C always passes pointers when passing arrays, so in theory, this should be the same